pkg/btf/validation.go

```diff
@@ -261,3 +262,54 @@ func validateSycall(kspec *v1alpha1.KProbeSpec, name string) error {
 
 	return nil
 }
+
+func GetSyscallsYaml(notMatch bool) (string, error) {
+	crd := `apiVersion: cilium.io/v1alpha1
+kind: TracingPolicy
+metadata:
+  name: "syscalls"
+spec:
+  kprobes:`
+
+	filter := `
+    selectors:
+    - matchBinaries:
+      - operator: "In"
+        values:
+        - "/krava"`
+
+	btfFile := "/sys/kernel/btf/vmlinux"
+
+	tetragonBtfEnv := os.Getenv("TETRAGON_BTF")
+	if tetragonBtfEnv != "" {
+		if _, err := os.Stat(tetragonBtfEnv); err != nil {
+			return "", fmt.Errorf("Failed to find BTF: %s", tetragonBtfEnv)
+		}
+		btfFile = tetragonBtfEnv
+	}
+
+	bspec, err := btf.LoadSpec(btfFile)
+	if err != nil {
+		return "", fmt.Errorf("BTF load failed: %v", err)
+	}
+
+	for _, key := range syscallinfo.SyscallsNames() {
+		var fn *btf.Func
+
+		key = "__x64_" + key
```

Comment:
I think we don't need the `__x64` prefix anymore, and omitting it should make things more portable wrt to ARM. CC: @mtardy.

Reply:
right, will change that

Reply:
yep exactly, please don't use :) thanks kkourt